assets/template-collection.css

```diff
@@ -447,7 +474,7 @@ a.active-facets__button:focus {
 
 .mobile-facets__wrapper {
   margin-left: 0;
-  margin-bottom: 3rem;
+  margin-bottom: 1rem;
 }
 
```

Comment:
Even though we cant scroll in the back anymore, I am still able to click on the header menu and other items:

https://screenshot.click/05-49-c7xxh-4exik.mp4

Reply:
Fixed - was a z-index issue.